socorro/external/boto/crashstorage.py

```diff
@@ -177,6 +178,8 @@ def build_s3_dirs(prefix, name_of_thing, crash_id):
     #--------------------------------------------------------------------------
     @staticmethod
     def do_save_raw_crash(boto_s3_store, raw_crash, dumps, crash_id):
+        if dumps is None:
+            dumps = MemoryDumpsMapping()
```

Comment:
should it perhaps, after that line, do something like `assert isinstance(dumps, MemoryDumpsMapping)`?